sources/packages/backend/apps/db-migrations/src/sql/Notifications/Create-notifications.sql

```diff
@@ -0,0 +1,36 @@
+CREATE TABLE IF NOT EXISTS notifications (
+    id SERIAL PRIMARY KEY,
+    -- Reference Columns
+    user_id INT REFERENCES users(id) ON DELETE CASCADE,
+    template_id VARCHAR(1000) NOT NULL,
+    message_id INT REFERENCES messages(id) ON DELETE CASCADE,
```

Comment:
If we are planning to use this table is a generic way `messages` could be enough, otherwise, we should consider having it as `notification_messages`, IMO.

Reply:
In same page on notification_messages.